src/index.html

```diff
@@ -2,16 +2,167 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-box">
+        <legend>
+          Personal information
+        </legend>
+        <label class="form-field">
+          Surname:
+          <input 
+          type="text"
+          name="Surname" 
+          autocomplete="off" 
+          required
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input 
+          type="text" 
+          name="Name" 
+          autocomplete="off" 
+          required
+          >
+        </label>
+        <label class="form-field">
+          How old are you?
+          <input 
+          type="number"
+          id="age" 
+          name="How old are you?"
+          min="1"
+          max="100"
+          value="12" 
+          autocomplete="off"
+          required
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input 
+          type="date"
+          id="start" 
+          name="trip-start" 
+          min="2000-01-01" 
+          max="2023-12-31" 
+          autocomplete="off"
+          required
+          >
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input 
+          type="checkbox" 
+          name="I accept the term of the agreement" 
+          autocomplete="off" 
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-box">
+        <legend>Registration</legend>
+        <label class="form-field">
+            E-mail:
+          <input 
+          type="email" 
+          name="E-mail" 
+          placeholder="email@example.com" 
+          autocomplete="off" 
+          required
+          >
+        </label>
+        <label class="form-field">
+            Password:
+          <input 
+          type="password" 
+          name="password" 
+          autocomplete="off" 
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-box">
+        <legend>
+            An interesting fact about you!
+        </legend>
+        <div class="form-field">
+            Do you love cats?
```

Comment:
```suggestion
        <div class="form-field">
           Do you love cats?
```

![image](https://user-images.githubusercontent.com/80700247/222440561-4753564f-9d9d-4421-8c11-c179165185dd.png)